src/Components/ImagesTable/ImageBuildErrorDetails.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+const useGetErrorReason = (err) => {
+    if (!err || !err.reason) {
+        return 'An unknown error occured';
+    }
+
+    if (err.details && err.details.reason) {
+        return err.details.reason;
+    }
+
+    return err.reason;
+};
+
+const FailureIcon = () => (
+    <Flex className="pf-u-align-items-baseline pf-m-nowrap">
+        <div className="pf-u-mr-sm">
+            <ExclamationCircleIcon className="error" />
+        </div>
+        <strong className='pf-u-danger-color-100'>Image build failed</strong>
+    </Flex>
+);
+
+const ErrorDetails = ({ status }) => {
+    if (!status || status.status !== 'failure') {
+        return <React.Fragment />;
+    }
+
+    const reason = useGetErrorReason(status.error);
+
+    return (
+        <div className='pf-u-mt-sm'>
+            <strong>Status</strong>
+            <FailureIcon />
+            <p className='pf-u-danger-color-100 pf-u-w-33-on-md'>
```

Comment:
Using PatternFly's `<Alert>` would also take care of this, although by default the `<Alert>` text isn't colored red (only the header) so we would still need some CSS.

Reply:
I guess it makes things slightly simpler. Will have a look

Reply:
Oh, good point... I agree that using the `<p>` tag is the way to go here, it is true to the mockup and consistent with the UUID alignment.